modules/rosa-aws-customer-managed-policies.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa-sts-about-iam-resources.adoc
+
+[id="rosa-aws-customer-managed-policies_{context}"]
+= Customer-managed policies
+ROSA users are able to attach customer-managed policies to the IAM roles required to run and maintain both ROSA (classic architecture) and ROSA clusters. This capability is not uncommon with AWS IAM roles.
+The ability to attach these policies to ROSA-specific IAM roles extends a ROSA cluster’s permission capabilities; for example, as a way to allow cluster components to access additional AWS resources that are otherwise not part of the ROSA-specific IAM policies.
+
+To ensure that any critical customer applications that rely on customer-managed policies are not modified in any way during cluster or role upgrades,  ROSA utilizes the ListAttachedRolesPolicies permission to determine whether a policy attached to a role is ROSA-specific or customer-managed. This information ensures that customer-managed policies are not impacted during cluster events, and allows Red Hat SREs to monitor both ROSA and customer-managed policies attached to ROSA specific IAM roles, enhancing their ability to troubleshoot any cluster issues more effectively.
```

Comment:
Adding permission here for transparency:
ListAttachedRolesPolicies and ListRolePolicies

ListAttachedRolesPolicies retrieves the list of permission policies from the role.
ListRolePolicies retrieves the list of inline policies from the role.